### PR TITLE
feat: add initial kokoro setup

### PR DESCRIPTION
**Pull request by burkedavison**

### CODE REVIEW

File: .kokoro/build.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```
Comment (jskeet) [inline .kokoro/build.sh:1]: Possibly rename to update-dotnet.sh?

File: .kokoro/build.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+go run ./cmd/generator update-repo \
```
Comment (jskeet) [inline .kokoro/build.sh:18]: If you set the environment variable in the Kokoro config (see previous PR, I can't remember the name offhand) you should be able to omit the image name.

**Review by jskeet — APPROVED** [review]
I'm sure it works as-is - just a couple of suggestions.

File: .kokoro/build.sh (hunk 1 above)
Comment (burkedavison) [reply .kokoro/build.sh]: Done

File: .kokoro/build.sh (hunk 2 above)
Comment (burkedavison) [reply .kokoro/build.sh]: Done

**Pull request merged**